psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -111,148 +114,162 @@ def run_sapt_dft(name, **kwargs):
+                                          conv=core.get_option("SAPT", "D_CONVERGENCE"),
+                                          Sinf=core.get_option("SAPT", "DO_IND_EXCH_SINF"))
+            hf_data.update(ind)
+            core.timer_off("SAPT(HF):ind")
+
+            dhf_value = hf_data["HF DIMER"] - hf_data["HF MONOMER A"] - hf_data["HF MONOMER B"]
+
+            core.print_out("\n")
+            core.print_out(print_sapt_hf_summary(hf_data, "SAPT(HF)", delta_hf=dhf_value))
+
+            data["Delta HF Correction"] = core.variable("SAPT(DFT) Delta HF")
+            sapt_jk.finalize()
+
+            del hf_wfn_A, hf_wfn_B, sapt_jk
 
-        # Move it back to monomer A
-        if (core.get_global_option('SCF_TYPE') == 'DF'):
-            core.IO.change_file_namespace(97, 'monomerB', 'dimer')
-
-        core.print_out("\n")
-        core.print_out("         ---------------------------------------------------------\n")
-        core.print_out("         " + "SAPT(DFT): delta HF Segment".center(58) + "\n")
-        core.print_out("\n")
-        core.print_out("         " + "by Daniel G. A. Smith and Rob Parrish".center(58) + "\n")
-        core.print_out("         ---------------------------------------------------------\n")
-        core.print_out("\n")
-
-        # Build cache
-        hf_cache = sapt_jk_terms.build_sapt_jk_cache(hf_wfn_A, hf_wfn_B, sapt_jk, True)
-
-        # Electrostatics
-        core.timer_on("SAPT(DFT):SAPT:elst")
-        elst = sapt_jk_terms.electrostatics(hf_cache, True)
-        hf_data.update(elst)
-        core.timer_off("SAPT(DFT):SAPT:elst")
-
-        # Exchange
-        core.timer_on("SAPT(DFT):SAPT:exch")
-        exch = sapt_jk_terms.exchange(hf_cache, sapt_jk, True)
-        hf_data.update(exch)
-        core.timer_off("SAPT(DFT):SAPT:exch")
-
-        # Induction
-        core.timer_on("SAPT(DFT):SAPT:ind")
-        ind = sapt_jk_terms.induction(hf_cache,
-                                      sapt_jk,
-                                      True,
-                                      maxiter=core.get_option("SAPT", "MAXITER"),
-                                      conv=core.get_option("SAPT", "D_CONVERGENCE"),
-                                      Sinf=core.get_option("SAPT", "DO_IND_EXCH_SINF"))
-        hf_data.update(ind)
-        core.timer_off("SAPT(DFT):SAPT:ind")
-
-        dhf_value = hf_data["HF DIMER"] - hf_data["HF MONOMER A"] - hf_data["HF MONOMER B"]
-
-        core.print_out("\n")
-        core.print_out(print_sapt_hf_summary(hf_data, "SAPT(HF)", delta_hf=dhf_value))
-
-        data["Delta HF Correction"] = core.variable("SAPT(DFT) Delta HF")
-        sapt_jk.finalize()
-
-        del hf_wfn_A, hf_wfn_B, sapt_jk
+        else:
+            wfn_A = hf_wfn_A 
+            wfn_B = hf_wfn_B
+            data["DFT MONOMERA"] = hf_data["HF MONOMER A"] 
+            data["DFT MONOMERB"] = hf_data["HF MONOMER B"] 
```

Comment:
In file https://github.com/psi4/psi4/blob/master/tests/sapt-compare/CMakeLists.txt, please add "quicktests" to the list like the below so that your changes go through the test suite.

`add_regression_test(sapt-compare "psi;sapt;cart;quicktests")`